runtime/syntax/man.vim

```diff
@@ -1,67 +1,34 @@
-" Vim syntax file
-" Language:	Man page
-" Maintainer:	SungHyun Nam <goweol@gmail.com>
-" Previous Maintainer:	Gautam H. Mudunuri <gmudunur@informatica.com>
-" Version Info:
-" Last Change:	2008 Sep 17
-
-" Additional highlighting by Johannes Tanzler <johannes.tanzler@aon.at>:
-"	* manSubHeading
-"	* manSynopsis (only for sections 2 and 3)
-
-" For version 5.x: Clear all syntax items
-" For version 6.x: Quit when a syntax file was already loaded
-if version < 600
-  syntax clear
-elseif exists("b:current_syntax")
+if exists("b:current_syntax")
   finish
 endif
 
-" Get the CTRL-H syntax to handle backspaced text
-if version >= 600
-  runtime! syntax/ctrlh.vim
-else
-  source <sfile>:p:h/ctrlh.vim
-endif
```

Comment:
Its useless as far as I can tell. If you remove line 122 `execute "silent! keepjumps %substitute,.\b,,g" of `autoload/man.vim` you'll see what it does. Man outputs backspaced text but in order to display it properly, we need to remove all those backspaces. This syntax file highlights them, whereas the substitution removes them. Thus if the substitution removes them, there is no need for the syntax file to be highlighting them. No idea why it was there before, man used to be piped into `col -b` which does the same thing, but doesn't work with other languages.